### PR TITLE
Fix tokens

### PR DESCRIPTION
**Pull request by fabriziovigevani**
- Updates Wrap Unrwap token icons (were a bit stretched)
- Header: Uses wrapable token name on BYOT garden
- Metrics: Uses wrapable token name on BYOT garden
- Decisions: Uses voting token as opposed to stake token

### CODE REVIEW

File: src/components/Metrics.js [hunk 1]
```diff
@@ -16,14 +16,16 @@ const Metrics = React.memo(function Metrics({
   const { layoutName } = useLayout()
   const compactMode = layoutName === 'small'
   const {
-    connectedGarden: { token },
+    connectedGarden: { wrappableToken, token },
   } = useGardens()
```
Comment (rperez89) [inline src/components/Metrics.js:20]: taking a look at the way that you implemented makes me wonder if we could identify on the Gardens provider if the dao is BYOT or normal and have new attr called `metricsToken` or `accountableToken` or whatever we want to call it and return it in the hook, that way we avoid `const metricsToken = wrappableToken || token` that we are doing on each component

**Review by rperez89 — COMMENTED** [review]
Looks good! just one small comment to improve a bit the logic

File: src/components/Metrics.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/Metrics.js:20]: Yeah i think we can do that when we do the general refactor of the logic which i was intending to do after reviewing the folder restructure PR.

**Comment by fabriziovigevani** [comment]
Closing in favor of #446

**Pull request closed without merging**